<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>862</width>
    <height>620</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>861</width>
     <height>571</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>821</width>
      <height>531</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>821</width>
      <height>531</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="live_tab">
     <attribute name="title">
      <string>Live Feed</string>
     </attribute>
     <widget class="QPushButton" name="recordButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>450</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QLabel" name="filepathText">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>601</width>
        <height>17</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>601</width>
        <height>17</height>
       </size>
      </property>
      <property name="text">
       <string>Saving video to: </string>
      </property>
     </widget>
     <widget class="QGroupBox" name="triggerGroup">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>130</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>131</height>
       </size>
      </property>
      <property name="title">
       <string>Record Upon:</string>
      </property>
      <widget class="QCheckBox" name="motionCheck">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Motion</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="subjectCheck">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Person, Pets</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="live_display">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>640</width>
        <height>360</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="setting_tab">
     <attribute name="title">
      <string>Setting</string>
     </attribute>
     <widget class="QGroupBox" name="emailGroup">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid gray;</string>
      </property>
      <property name="title">
       <string>SMTP Setup (for email alerts)</string>
      </property>
      <widget class="QTextEdit" name="email_txtbox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="email_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
      <widget class="QLabel" name="password_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="password_txtbox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="smtp_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>SMTP:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="smtp_txtbox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="testButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Send Test Email</string>
       </property>
      </widget>
      <widget class="QLabel" name="port_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="port_txtbox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>190</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Setup</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
